client/extensions/woocommerce/app/settings/payments/index.js

```diff
@@ -22,14 +29,30 @@ import SettingsPaymentsOnSite from './payments-on-site';
 class SettingsPayments extends Component {
 
 	static propTypes = {
+		isSaving: PropTypes.bool,
 		site: PropTypes.shape( {
 			slug: PropTypes.string,
 		} ),
 		className: PropTypes.string,
 	};
 
+	onSave = () => {
+		const { translate } = this.props;
+
+		const successAction = successNotice(
+			translate( 'Payment settings saved.' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **27** times:
`translate( 'Payment Settings' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).